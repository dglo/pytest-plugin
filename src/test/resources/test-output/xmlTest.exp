tsts=3
errs=2
fail=0
time=0.001
xml=true
