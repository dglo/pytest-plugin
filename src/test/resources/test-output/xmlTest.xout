<testsuite errors="2" failures="0" name="unittest.TestSuite" tests="3" time="0.001">
  <testcase classname="__main__.xmlTest" name="testExcBlank" time="0.0005">
    <error type="exceptions.Exception">
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.5/lib/python2.5/unittest.py", line 260, in run
    testMethod()
  File "subtst/xmlTest.py", line 13, in testExcBlank
    raise Exception()
    </error>
  </testcase>
  <testcase classname="__main__.xmlTest" name="testExcText" time="0.0001">
    <error type="exceptions.Exception">Something
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.5/lib/python2.5/unittest.py", line 260, in run
    testMethod()
  File "subtst/xmlTest.py", line 10, in testExcText
    raise Exception("Something")
    </error>
  </testcase>
  <testcase classname="__main__.xmlTest" name="testGood" time="0.0000"></testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
