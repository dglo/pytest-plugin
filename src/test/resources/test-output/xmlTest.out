testExcBlank (__main__.xmlTest) ... ERROR
testExcText (__main__.xmlTest) ... ERROR
testGood (__main__.xmlTest) ... ok

======================================================================
ERROR: testExcBlank (__main__.xmlTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "subtst/xmlTest.py", line 13, in testExcBlank
    raise Exception()
Exception

======================================================================
ERROR: testExcText (__main__.xmlTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "subtst/xmlTest.py", line 10, in testExcText
    raise Exception("Something")
Exception: Something

----------------------------------------------------------------------
Ran 3 tests in 0.001s

FAILED (errors=2)
